### DISCUSSION TITLE
--multi: select-all but stdout with same order

### DISCUSSION THREAD

**Discussion by LintaoAmons** [junegunn/fzf#3584] [Q&A] [answered]
<img width="1434" alt="image" src="https://github.com/junegunn/fzf/assets/95092244/26ba4491-db70-4148-a739-35d9852e5f1a">

How can I keep the output in the same order?

**Comment by junegunn** [comment, marked as answer]
The default layout displays the items from the bottom up

```sh
seq 100 | fzf
```

And `select-all` respects the order.

If you don't like it, you can use `reverse` or `reverse-list` layout so that the display order matches the order of the output

```sh
seq 100 | fzf --layout=reverse
seq 100 | fzf --layout=reverse-list
```

or still use the default display order and reverse the output list after fzf using `tac` (or `tail -r` on macOS)

```sh
seq 100 | fzf | tac
```
